catalog.html

```diff
@@ -0,0 +1,276 @@
+      </header>
+
+      <section class="catalog-products">
+        <h2 class="visually-hidden">Перечень гостиниц с фильтрами</h2>
+        <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Инфраструктура:</legend>
+            <ul class="catalog-filter-list">
+
+                <li class="catalog-filter-item">
+                  <label class="control">
+                    <input class="control-input" type="checkbox" name="pool" checked>
+                    Бассейн
+                  </label>
+                </li>
+                <li class="catalog-filter-item">
+                  <label class="control">
+                    <input class="control-input" type="checkbox" name="parking" checked>
+      
+                    Парковка
+                  </label>
+                </li>
+                <li class="catalog-filter-item">
+                  <label class="control">
+                    <input class="control-input" type="checkbox" name="Wi-Fi">
+      
+                    Wi-Fi
+                  </label>
+                </li>
+            </ul>
+          </fieldset>
+
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Тип жилья:</legend>
+            <ul class="catalog-filter-list">
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="product-group" value="Hotel" checked>
+
+                  Гостиница 
+                </label>
+              </li>
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="product-group" value="Motel">
+
+                  Мотель
+                </label>
+              </li>
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="product-group" value="Apartments">
+
+                  Апартаменты
+                </label>
+              </li>
+            </ul>
+          </fieldset>
+
+          <div class="catalog-filter-group">
+            <p class="catalog-filter-title">Стоимость,₽:</p>
+  
+            <div class="catalog-filter-interval">
+              <label class="catalog-filter-label">
+                От <input class="catalog-filter-input-text" type="text" name="min-price" value="0">
+              </label>
+              <label class="catalog-filter-label">
+                До <input class="catalog-filter-input-text" type="text" name="max-price" value="9000">
+              </label>
+              <div class="catalog-filter-price"></div>
+            </div>
+          </div>
+          <button class="button" type="submit">Применить</button>
+          <button class="button" type="submit">Сбросить</button>
```

Comment:
```suggestion
          <button class="button" type="reset">Сбросить</button>
```
так как `type="reset"` сбрасывает значения полей формы к значениям по умолчанию.